collector/collector/report_handler.py

```diff
@@ -110,6 +122,35 @@ def before_request():
     )
 
 
+def process_multipart_payload():
+    file_path = tempfile.NamedTemporaryFile(dir=ATTACHMENT_QUARANTINE_FOLDER, prefix="{}".format(time.time())).name
+    file_handler = request.files.get('payload', None)
+    if file_handler is not None:
+        file_handler.save(file_path)
+        return file_path, file_handler.mimetype
+    return None, request.content_type[:19]
```

Comment:
I would prefer to see `request.content_type[:19]` wrapped in a function call so it is more clear what substring is being extracted.

Reply:
Agree